src/factoids.coffee

```diff
@@ -76,8 +77,15 @@ module.exports = (robot) ->
       msg.reply 'Not a factoid'
 
   robot.respond /factoids?/i, (msg) =>
-    url = process.env.HUBOT_BASE_URL or "http://not-yet-set/"
-    msg.reply "#{url.replace /\/$/, ''}/#{robot.name}/factoids"
+    all = @factoids.getAll()
```

Comment:
Rather than replacing the functionality of this command, let's add a separate (new) command.